rest_framework/fields.py

```diff
@@ -53,14 +53,30 @@ def is_simple_callable(obj):
     """
     True if the object is a callable that takes no arguments.
     """
+    if not hasattr(inspect, 'signature'):
+        return py2k_is_simple_callable(obj)
```

Comment:
Do you mean `six.PY3`? There is actually a problem with python 3.2, which doesn't have `signature`.